orb.yml

```diff
@@ -16,7 +16,7 @@ commands:
       - save_cache:
           key: buildevents-v0.14.0{{ .Environment.BUILDEVENTS_CACHE_VERSION }}
           paths:
-            - /tmp/be
+            - ~/project/bin
```

Comment:
I'm slightly concerned about the change of `/tmp/be` here. Is there a specific reason you need that other than "it might break sometimes"? We're still using `/tmp/buildevents` for timestamp tracking, so it feels like we've only solved 50% of the problem if it's really a problem.